README.md

```diff
@@ -0,0 +1,335 @@
+# virus-scanner-service
+
+Microservice to perform a virus scan on uploaded files.
+
+This service listens for delta notifications about new files and scans
+those for viruses.
+
+[![Project Status: WIP – Initial development is in progress](https://www.repostatus.org/badges/latest/wip.svg)](https://www.repostatus.org/#wip)
+
+## Getting started
+
+Prerequisites:
+- [file-service](https://github.com/mu-semtech/file-service)
+- [mu-authorization](https://github.com/mu-semtech/mu-authorization)
+- [delta-notifier](https://github.com/mu-semtech/delta-notifier)
+
+### Adding the service to your stack
+
+Add the following snippet to your `docker-compose.yml` to include this
+service in your project.
+
+```yaml
+version: '3.4'
+
+services:
+  virus-scanner:
+    build: https://github.com/peternowee/virus-scanner-service.git#v0.0.8
+    links:
+      - database:database
+    environment:
+      VIRUS_SCANNER_CLAMD_USER: # "root"
+    volumes:
+      - ./data/files:/share
+      - type: volume
```

Comment:
Two questions here:
1. do we need to persist these signatures? 
2. if we do, can't this not be in a simple mounted volume. (It's one of these implicit assumptions of the stack we DON'T work with docker-volumes)

Reply:
>     1. do we need to persist these signatures?

If you don't, you risk getting rate-limited for downloading the signature database too often:

```
WARNING: Tue Oct 24 18:08:37 2023 -> Can't download daily.cvd from https://database.clamav.net/daily.cvd
WARNING: Tue Oct 24 18:08:37 2023 -> FreshClam received error code 429 from the ClamAV Content Delivery Network (CDN).
WARNING: Tue Oct 24 18:08:37 2023 -> You are on cool-down until after: 2023-10-25 14:48:57
WARNING: Tue Oct 24 18:08:38 2023 -> Can't download main.cvd from https://database.clamav.net/main.cvd
WARNING: Tue Oct 24 18:08:38 2023 -> FreshClam received error code 429 from the ClamAV Content Delivery Network (CDN).
WARNING: Tue Oct 24 18:08:38 2023 -> You are on cool-down until after: 2023-10-25 14:48:57
Tue Oct 24 18:08:38 2023 -> This means that you have been rate limited by the CDN.
Tue Oct 24 18:08:38 2023 ->  1. Run FreshClam no more than once an hour to check for updates.
Tue Oct 24 18:08:38 2023 ->     FreshClam should check DNS first to see if an update is needed.
Tue Oct 24 18:08:38 2023 ->  2. If you have more than 10 hosts on your network attempting to download,
Tue Oct 24 18:08:38 2023 ->     it is recommended that you set up a private mirror on your network using
Tue Oct 24 18:08:38 2023 ->     cvdupdate (https://pypi.org/project/cvdupdate/) to save bandwidth on the
Tue Oct 24 18:08:38 2023 ->     CDN and your own network.
Tue Oct 24 18:08:38 2023 ->  3. Please do not open a ticket asking for an exemption from the rate limit,
Tue Oct 24 18:08:38 2023 ->     it will not be granted.
```